### ISSUE TITLE
Not signing in 

### ISSUE THREAD

**Issue by abhishek-kira** [LeetCode-OpenSource/vscode-leetcode#773]
## 🐛 Bug Report

Its not signing in through the cookie option
## To Reproduce

Steps to reproduce the behavior.

## Extension Output

Paste here the LeetCode extension log from output channel.

Guidance: Press `Ctrl+Shift+U`, and toggle the channel to `LeetCode`.

## Your Environment

- *os*:
- *extension settings*:
- *nodejs version*:
- *vscode version*:
- *extension version*:

**Comment by abhishek-kira** [comment]
Its not signing through the cookie option

**Comment by nuynait** [comment]
+1

Not able to sign in using the cookie today. I was able to do this a couple of weeks before.

**Comment by nuynait** [comment]
I have removed `~/.lc` and `~/.vscode/extensions/leetcode.vscode-leetcode-x.xx.x`

Then relaunch visual studio, install leetcode and perform login again. This time I no longer have issues I had above.

**Comment by nuynait** [comment]
Also I noticed that I was using Firefox to grab the token but this time I used Chrome. The token I get from Chrome seems to be different from what I got from Firefox. This might be the reason I am able to login again.

**Comment by dtmw8z** [comment]
> Also I have noticed that I was using Firefox to grab the token but this time I used Chrome. The token I get from Chrome seems to be different from what I got from Firefox. This might be the reason I am able to login again.

I agree. I used firefox before and got the exactly same issue. But then using edge solved the issue

**Comment by foxbit19** [comment]
Same here. Firefox's cookie is different. I've used chrome and I was able to login
